its/core-it-support/maven-it-helper/src/main/java/org/apache/maven/it/AbstractMavenIntegrationTestCase.java

```diff
@@ -260,6 +261,17 @@ protected Verifier newVerifier(String basedir, boolean debug) throws Verificatio
     protected Verifier newVerifier(String basedir, String settings, boolean debug) throws VerificationException {
         Verifier verifier = new Verifier(basedir);
 
+        // try to get jacoco arg from command line if any then use it to start IT to populate jacoco data
```

Comment:
I just leave it but it's not activated in the build as it's definitely too slow.